pyhive/tests/sqlalchemy_test_case.py

```diff
@@ -64,7 +64,8 @@ def test_reflect_include_columns(self, engine, connection):
         """When passed include_columns, reflecttable should filter out other columns"""
         one_row_complex = Table('one_row_complex', MetaData(bind=engine))
         engine.dialect.reflecttable(
-            connection, one_row_complex, include_columns=['int'], exclude_columns=[])
+            connection, one_row_complex, include_columns=['int'],
```

Comment:
thanks for the fixes ! 
https://docs.sqlalchemy.org/en/13/core/reflection.html
looks like specifying resolve_fks=True is optional as True is a default value.
Let's remove  resolve_fks=True from the tests, otherwise PR looks good.

Reply:
I tried to push a new commit, but I get the following:

pyhive/tests/sqlalchemy_test_case.py:67: in test_reflect_include_columns
    connection, one_row_complex, include_columns=['int'], exclude_columns=[])
E   TypeError: reflecttable() missing 1 required positional argument: 'resolve_fks'

I think that the doc that you pointed out might not be the reflecttable() that we are looking into, since the resolve_fks seems a positional parameter.

Reply:
For example in https://kite.com/python/docs/sqlalchemy.engine.Dialect.reflecttable it is listed as positional parameter, maybe a doc issue on the sqlalchemy side?

Reply:
@bkyryliuk https://github.com/sqlalchemy/sqlalchemy/commit/a690ec082451ca0636fb45a0d9773a240ae30373 should explain it. I think that in this case we are not hitting reflection.py, but default.py (in which the parameter is required and doesn't have any default).